azurelinuxagent/ga/exthandlers.py

```diff
@@ -822,7 +823,7 @@ def download(self):
         self.report_event(message="Download succeeded", duration=duration)
 
     def initialize(self):
-        self.logger.info("Initialize extension directory")
+        self.logger.info("Initialize extension")
```

Comment:
Since you're updating this, can we add the extension name here? The message seems ambiguous without it.

Reply:
yup